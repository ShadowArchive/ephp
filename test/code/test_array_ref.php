<?php

$a = [ "name" => "iq",
       "attrs" => [ "id" => 1,
                    "type" => "result" ],
       "children" => [] ];
$attrs = &$a["attrs"];

var_dump($a);
var_dump($attrs);

