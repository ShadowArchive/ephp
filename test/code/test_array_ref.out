array(3) {
  ["name"]=>
  string(2) "iq"
  ["attrs"]=>
  &array(2) {
    ["id"]=>
    int(1)
    ["type"]=>
    string(6) "result"
  }
  ["children"]=>
  array(0) {
  }
}
array(2) {
  ["id"]=>
  int(1)
  ["type"]=>
  string(6) "result"
}
